maintainers-info-report/Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+@Library('pipeline-library@pull/348/head') _
+
+pipeline {
+	triggers {
+    // Execute once per hour
+		cron('@hourly')
+	}
+	options {
+		// This pipeline takes 30-40 minutes to execute
+		timeout(time: 1, unit: 'HOURS')
+    lock(resource: 'infra-reports-maintainers-info', inversePrecedence: true)
+	}
+	agent {
+    kubernetes {
+      yamlFile 'JenkinsAgentPodTemplate.yaml'
+    }
+  }
+	stages {
+    stage('Generate Maintainers Report') {
+      environment {
+        JIRA_AUTH = credentials('jiraAuth')
+      }
+      steps {
+        dir('maintainers-info-report') {
+          sh 'bash ./maintainers-info-report.sh > maintainers-info-report.json'
```

Comment:
```suggestion
          sh '''
          JIRA_AUTH="${JIRA_AUTH_USR}:${JIRA_AUTH_PSW}"
          bash ./maintainers-info-repor.sh > maintainers-info-report.json
          '''
```

Reply:
(otherwise the job wouldn't have been able to succeed)

Reply:
~~I'm surprised, in the previous version it was called with:~~
```bash
sh 'docker run -e JIRA_AUTH="${JIRA_AUTH_USR}:${JIRA_AUTH_PSW}" maintainers-info-report > maintainers-info-report.json'
```
~~and I don't see any modification to this script...~~
Nevermind, the credentials mechanism creates the two variables JIRA_AUTH_USR & JIRA_AUTH_PSW used in the script.

Reply:
It's a username credential precisely: https://github.com/jenkins-infra/kubernetes-management/blob/0b1757daf0b8f64e8d8861a2b7e81270db0d249c/config/ext_jenkins-infra-jobs.yaml#L170-L174